### DISCUSSION TITLE
Práctica de CSS lista! :D

### DISCUSSION THREAD

**Discussion by marco-cortes** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#354] [Show and tell]
![landing-page](https://user-images.githubusercontent.com/84252964/157997200-e9391427-34f9-4a5f-9a94-9972e551249a.png)

Les dejo mi repo por si les sirve algo: https://github.com/marco-cortes/vaccination-landing-page

**Comment by OscarPedroza** [comment]
Excelente trabajo compañero!

**Comment by DanielGalindoRios** [comment]
Excelente!!

Te quedo muy bien. Vere tu codigo para ver unas cosas que no me quedan.

Gracias por compartir y es un trabajo excelente

**Comment by JuanSalvador73** [comment]
Te quedó genial :D
